docs/content/ai_integrations/anthropic.md

```diff
@@ -1,6 +1,6 @@
 # Anthropic
 
-`superduper` allows users to work with `anthropic` API models. The key integration is the integration 
+`Superduper` allows users to work with `anthropic` API models. The key integration is the integration 
```

Comment:
No this is the package.

Reply:
Noted. The package will be package. I will keep the `superduper` as it is where it mentions the package.